src/components/quiz/UserDetailsForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+interface User {
+  name: string | null;
+  displayPic: string | null;
+}
+
+export const UserDetailsForm: React.FunctionComponent = ({status,setStatus}: any) => {
+
+  const [userDetails, setUserDetails] = useState<User>({
+    name: null,
+    displayPic:null,
+  });
+  
+
+  function onSubmit(event: any) {    
```

Comment:
Prefer arrow functions
```suggestion
  const onSubmit = (event: any) => {    
```

Reply:
Don't use `any`! Everything has a type, you just need to find out what it is!
```suggestion
  const onSubmit: React.FormEventHandler<HTMLFormElement> = (event) => {
```